src/main/java/com/educare/unitylend/controller/BorrowRequestController.java

```diff
@@ -44,10 +73,19 @@ ResponseEntity<Boolean> createBorrowRequest(@RequestBody BorrowRequest borrowReq
      */
     @GetMapping("/get-borrow-request-by-user-id/{userId}")
     ResponseEntity<List<BorrowRequest>> getBorrowRequestForUserId(@PathVariable String userId) throws ControllerException {
```

Comment:
There is no need to put null checks separately 
Please validate if` @PathVariable(required = false)` helps. 

Make the change in all the controller methods